spiffe.go

```diff
@@ -93,3 +101,12 @@ func GetURINamesFromPEM(encodedCertificate string) (uris []string, err error) {
 
 	return GetURINamesFromCertificate(cert)
 }
+
+// FuriNamesFromPEM retrieve URIs from the SAN extension of the reader containing a PEM-encoded X.509 certificate
+func FuriNamesFromPEM(f io.Reader) (uris []string, err error) {
```

Comment:
Interestingly I actually had it before as `FgetURINamesFromPEM` but thought the `get` was redundant, however I'll update that. Thanks for letting me know.